[2024-06-27T16:28:43.649+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-27T16:28:43.669+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: preprocess_data.wait_for_download manual__2024-06-27T16:28:40.949709+00:00 [queued]>
[2024-06-27T16:28:43.675+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: preprocess_data.wait_for_download manual__2024-06-27T16:28:40.949709+00:00 [queued]>
[2024-06-27T16:28:43.675+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-06-27T16:28:43.684+0000] {taskinstance.py:2330} INFO - Executing <Task(ExternalTaskSensor): wait_for_download> on 2024-06-27 16:28:40.949709+00:00
[2024-06-27T16:28:43.694+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=66) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-27T16:28:43.695+0000] {standard_task_runner.py:63} INFO - Started process 72 to run task
[2024-06-27T16:28:43.697+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'preprocess_data', 'wait_for_download', 'manual__2024-06-27T16:28:40.949709+00:00', '--job-id', '11', '--raw', '--subdir', 'DAGS_FOLDER/preprocess_data.py', '--cfg-path', '/tmp/tmpjf0od4s1']
[2024-06-27T16:28:43.699+0000] {standard_task_runner.py:91} INFO - Job 11: Subtask wait_for_download
[2024-06-27T16:28:43.711+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:195: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2024-06-27T16:28:43.734+0000] {task_command.py:426} INFO - Running <TaskInstance: preprocess_data.wait_for_download manual__2024-06-27T16:28:40.949709+00:00 [running]> on host dc4243757e06
[2024-06-27T16:28:43.791+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='preprocess_data' AIRFLOW_CTX_TASK_ID='wait_for_download' AIRFLOW_CTX_EXECUTION_DATE='2024-06-27T16:28:40.949709+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-06-27T16:28:40.949709+00:00'
[2024-06-27T16:28:43.792+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-27T16:28:43.801+0000] {baseoperator.py:400} WARNING - ExternalTaskSensor.execute cannot be called outside TaskInstance!
[2024-06-27T16:28:43.802+0000] {external_task.py:258} INFO - Poking for tasks ['download_stock_data'] in dag download_data on 2024-06-27T16:28:40.949709+00:00 ... 
[2024-06-27T16:28:43.802+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/utils/session.py:79: DeprecationWarning: This method is deprecated and will be removed in future.
  return func(*args, session=session, **kwargs)

[2024-06-27T16:29:43.892+0000] {external_task.py:258} INFO - Poking for tasks ['download_stock_data'] in dag download_data on 2024-06-27T16:28:40.949709+00:00 ... 
[2024-06-27T16:30:46.008+0000] {external_task.py:258} INFO - Poking for tasks ['download_stock_data'] in dag download_data on 2024-06-27T16:28:40.949709+00:00 ... 
[2024-06-27T16:31:46.079+0000] {external_task.py:258} INFO - Poking for tasks ['download_stock_data'] in dag download_data on 2024-06-27T16:28:40.949709+00:00 ... 
[2024-06-27T16:32:49.532+0000] {external_task.py:258} INFO - Poking for tasks ['download_stock_data'] in dag download_data on 2024-06-27T16:28:40.949709+00:00 ... 
[2024-06-27T16:33:39.923+0000] {local_task_job_runner.py:124} ERROR - Received SIGTERM. Terminating subprocesses
[2024-06-27T16:33:39.962+0000] {process_utils.py:132} INFO - Sending 15 to group 72. PIDs of all processes in the group: [72]
[2024-06-27T16:33:39.965+0000] {process_utils.py:87} INFO - Sending the signal 15 to group 72
[2024-06-27T16:33:39.975+0000] {taskinstance.py:2611} ERROR - Received SIGTERM. Terminating subprocesses.
[2024-06-27T16:33:40.033+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
